Packs/LogRhythm/Integrations/integration-LogRhythm.yml

```diff
@@ -1332,21 +1067,21 @@ script:
         };
     }
 
-
     function fetchIncidents(){
 
         var dataObj = {
             Username: params.Credentials.identifier,
             Password: params.Credentials.password
         };
 
-        var lastRun = getLastRun();
+        var lastRun = getLastRun() + tz;
```

Comment:
i dont think that will work, lastrun is an object, i think you meant to do something like `lastRun.lastTime`